crawl-ref/source/item-name.cc

```diff
@@ -2914,9 +2917,7 @@ bool is_useless_item(const item_def &item, bool temp, bool ident)
         switch (item.sub_type)
         {
         case SCR_TELEPORTATION:
-            return you.stasis()
-                   || crawl_state.game_is_sprint()
-                   || temp && player_in_branch(BRANCH_GAUNTLET);
```

Comment:
Thanks for the review!

I removed the check for ?tele in Gauntlet and a few others, because `cannot_read_item_reason()` already has them. I've added a comment to the relevant part in `is_useless_item()` to make it more clear.